### ISSUE TITLE
Milestone 1 Feedback

### ISSUE THREAD

**Issue by tonyliang19** [DSCI-310/dsci-310-group-07#9]
Please read below feedbacks on things that you could've improve on for future milestones, **NOTE** if these are not fixed, marks will be deducted. If you have any questions on these feedbacks, please feel free to drop-in to my office hour or during lecture time

# Milestone 1 Feedback Group 7

### README.md

- [x] Instructions not clear to reproduce project, consider adding command for `git clone`, and how to `cd` **-1**

- [x] List of dependecies is not listed here, we explicitly asked to list it in README.md **-1**

### Analysis

It is fine to include both `.Rmd` or `.ipynb` files, but remember to provide its corresponding instruction to reproduce it (See below at [Reproducibility](#reproducibility))

- [x] Fix markdown table render in your description, otherwise it just looks like a huge chunk of text that do not belong to the analysis and makes it hard to follow. **-1**
### Dockerfile

- [x] Extra package `cowsay`, please only include packages that you are using in the analysis, as this part of designing reproducible environment with minimal requirements runnable. **-1**

#### Workflow

- [x] Only triggers, when changes are being maded on **Dockerfile**, and not on any changes pushed / PR to main. **-1**

#### Project/Milestones/Issues

- [x] No GitHub Milestones found **-5**
- [x] No GitHub Project found **-5**

#### Flow Version Control

Please use meaning full names for your branches later, `alternate/test/todo` is not enough, be more specific like:
- `fix-<feature>`
- `<feature-x>`

For future milestones this will be more stricly graded
### Project File and Directory Structure

- [x] Extra .DS_Store (use .gitignore) **-1**

### Reproducibility

Please provide more detailed solution for different OS systems, your solution should take care of Windows/Mac/Linux users, **FYI**, Windows is failing 

- [x] Failed to reproduce analysis, check the path your trying to mount into the container, and did not provide instructions on which of **.ipynb** or **.Rmd** file should run. **-10**

- [x] Environment is only considered for R and not for Jupyter **-2**

**Comment by harborzhang2000** [comment]
delete the unused cowsay package from Dockerfile.

**Comment by uliaLiao** [comment]
Fix the other 8 except for the project (it is private so invisible 😢 )

**Comment by tonyliang19** [comment]
This is very fast from you guys, good progress 👍🏼

**Comment by tonyliang19** [comment]
One more suggestion is you could link issues/PR corresponding to the task to be solved from above like #12, this helps to find how, where you take care of the task
